app/webpacker/stylesheets/application.scss

```diff
@@ -11,3 +11,13 @@ $button-colour: #00823b;
 .text-decoration-underline-dotted {
   text-decoration: underline dotted;
 }
+
+.course-table {
+  .govuk-table__head {
+    border-bottom: 2px solid $govuk-input-border-colour;
```

Comment:
I've been thinking of dropping this style in favour of consistency with design system. What do you think?

Reply:
Yep, I don't think it adds any value to the UI

Reply:
Agree. Let's leave this out and we can update the others later.